aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -64,16 +65,28 @@ module aptos_framework::account {
         new_public_key: vector<u8>,
```

Comment:
Not related to this PR, but can we clarify what some of the fields in `RotationProofChallenge` above are? (I can only add this comment here, but see the struct definition a few lines above it.)

I would add the following comments:
```
sequence_number: u64, // the sequence number of the account whose key is being rotated
originator: address, // the address of the account whose key is being rotated
current_auth_key: address, // the current authentication key of the account whose key is being rotated
new_public_key: vector<u8>, // the public key that the account will be rotated to 
```
Note that right now, we have `// originating address` as a comment for `current_auth_key` which is not even correct, no?